lighthouse-core/scripts/compare-runs.js

```diff
@@ -120,12 +126,15 @@ function round(value) {
  * @param {number} total
  * @return {string}
  */
-function getProgressBar(i, total = argv.n * argv.urls.length) {
+function getProgressBar(i, total = argv.n * (argv.urls || []).length) {
```

Comment:
no way to do required arrays?

Reply:
> no way to do required arrays?

But I think we only want a required array of `urls` if doing `--collect`? [`yargs.implies()`](https://github.com/yargs/yargs/blob/e2d9e9364099fe1f1992d4beaff1cbd56ad3df28/docs/api.md#impliesx-y) can do that, I believe, but it's a little too indirect for the typescript type to infer off of. It might benefit the CLI experience, but a JS check or a default will still have to go somewhere.

I was trying to leave everything mostly untouched so it would all be recognizable when you were next back in here, but happy to commit any suggestions